src/styles/components/shop.scss

```diff
@@ -0,0 +1,21 @@
+.shop {
+  justify-items: center;
+  margin: 0 auto;
+  padding: 50px 40px;
+  width: 1024px;
```

Comment:
You should delete it to pass the last test. As you can see in the test there is 1200px but this CSS prop doesn`t allow it
![image](https://github.com/mate-academy/layout_catalog/assets/64101898/2b6139ec-9afe-404f-916c-4345ecf11ddd)

Reply:
Looks like i hardcoded because of two days debugging.  I tried several variants but the test is still impossible to pass